### ISSUE TITLE
Not possible to use multiple recipients

### ISSUE THREAD

**Issue by rbuffi** [ix-ai/grafana-email#3]
Hello, I'm trying to sent my grafana reports to multiple recpients by using comma separated mail addresses as value for the SMTP_TO environment variabele but this is not working. Can you help me out defining multiple recipients? Thanks in advance.

**Comment by tlex** [comment]
This functionality is currently not implemented. The python [smtplib library]() supports it, but it's currently [called](https://github.com/ix-ai/grafana-email/blob/master/grafana-email/grafana-email.py#L147) with a string, not a list.

I could imagine adding a function to parse the string from the environment variable on [#L32](https://github.com/ix-ai/grafana-email/blob/master/grafana-email/grafana-email.py#L32), however this will take a bit of time (probably until the weekend).

In the meanwhile, as a workaround, you can call the script multiple times, for every destination address you have.

**Comment by rbuffi** [comment]
That would be great. I got a large list of recipients so for now I will sent the reports to myself and redistribute. Let me know when the function is implemented so I can test it for you. Thanks again!

**Comment by tlex** [comment]
I have made the changes, but haven't merged them yet to master.

A new image already has been built, if you want to try it out: `registry.gitlab.com/ix.ai/grafana-email:dev-branch`. I have tested it successfully with two recipients.

Personally, I still recommend as the more elegant solution to use a mailing list as recipient - this way there's no need to change the systemd timer / cron job every time someone wants to be added / removed.

Edit: forgot to mention - just use a space separated list for `SMTP_TO` (such as `email1@domain email2@domain2 email12@domain14`)
